pkg/authorization/interfaces.go

```diff
@@ -0,0 +1,31 @@
+package authorization
+
+import (
+	"net/http"
+
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
+)
+
+// Policy is a Rule policy name
+type Policy string
+
+const (
+	AllowPolicy = Policy("ALLOW")
+	DenyPolicy  = Policy("DENY")
+	AuthPolicy  = Policy("AUTH")
+	SkipPolicy  = Policy("SKIP")
```

Comment:
Is this a user accessible policy? Does it need to be public?

Reply:
Good call -- it is just for internal purposes, I'll make it private.